app/v1/support/IfsResponseMappingSupport.scala

```diff
@@ -24,7 +25,7 @@ import v1.models.outcomes.ResponseWrapper
 trait IfsResponseMappingSupport {
```

Comment:
As you've changed the generics to `DownstreamError`, should this class be specific to IFS? If not, better to change all references to IFS to `Downstream`

Reply:
Yes